### PR TITLE
perfomance optimization for domUtils.isElementInDocument (part of #424)

### PR DESCRIPTION
**Pull request by miherlosev**
/cc @AlexanderMoskovkin @LavrovArtem 
see http://jsperf.com/document-contains-versus-iterative-algorith/2

### CODE REVIEW

File: src/client/utils/dom.js [hunk 1]
```diff
@@ -334,17 +347,12 @@ export function isDomElement (el) {
...
 
 export function isElementInDocument (el, currentDocument) {
-    var doc        = currentDocument || document;
-    var curElement = el.parentNode;
+    var doc = currentDocument || document;
 
-    while (curElement) {
-        if (curElement === doc)
-            return true;
+    if (doc.contains)
```
Comment (AlexanderMoskovkin) [inline src/client/utils/dom.js]: Not all of browsers support `document.contains` (for example, IE11), but as I've found most of browsers support `document.body.contains`. Maybe we can use it here?

Comment (AlexanderMoskovkin) [inline src/client/utils/dom.js]: or `document.documentElement.contains`

Comment (miherlosev) [inline src/client/utils/dom.js]: We cannot use `document.body.contains`.
Inside functions `appendChild`, `insertBefore` we can use this function for `head` element.
see https://github.com/DevExpress/testcafe-hammerhead/blob/master/src/client/sandbox/node/element.js#L201

Comment (miherlosev) [inline src/client/utils/dom.js]: Ok. I will research for using `document.documentElement`.

**Comment by miherlosev** [comment]
There is a potential problem with `document.documentElement.contains`.
In some browsers (IE, FF) `document.documentElement` for iframe don't exists right after iframe adding.
In hammerhead we don't use `isElementInDocument` for `document` without `documentElement`.
If we don't use this scenario in `testcafe` we can just use `document.documentElement.contains`.

**Comment by AlexanderMoskovkin** [comment]
As i see there is no problem with this.

**Comment by AlexanderMoskovkin** [comment]
lgtm

**Comment by LavrovArtem** [comment]
lgtm

**Pull request merged**